srcpkgs/setzer/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'setzer'
+pkgname=setzer
+version=0.3.2
+revision=1
+wrksrc=Setzer-${version}
+build_style=meson
+hostmakedepends="gettext"
+makedepends="python3-gobject-devel"
+depends="libpoppler89 texlive-core gspell python3-xdg python3-gobject gtksourceview4"
```

Comment:
`virtual?tex`

Reply:
>Theoretically, someone could use texlive-bin with this package and expect everything to work, correct?

I guess you're correct. Changes have been pushed

Reply:
It seems that `virtual?tex` is giving problem on arm (`aarch64`, `armv7l`, `armv6l-musl` and `aarch64-musl`), travis log says:

```
=> texlive-20200406_4: installing host dependencies: pkg-config-0.29.2_2 perl-5.30.3_1 lua52-BitOp-1.0.2_3 texinfo-6.7_1 texlive-20200406_4 cross-i686-linux-musl-0.33_3 ...

=> texlive-20200406_4: installing target dependencies: cairo-devel-1.16.0_2 freetype-devel-2.10.2_1 gd-devel-2.3.0_2 graphite-devel-1.3.14_1 gmp-devel-6.2.0_1 harfbuzz-devel-2.7.2_2 icu-devel-67.1_1 libpaper-devel-1.1.28_1 libpng-devel-1.6.37_1 mpfr-devel-4.0.2_1 poppler-devel-20.09.0_1 pixman-devel-0.38.4_1 libteckit-devel-2.5.10_1 zlib-devel-1.2.11_3 zziplib-devel-0.13.71_1 libXaw-devel-1.0.13_2 ...

=> texlive-20200406_4: running do-fetch hook: 00-distfiles ...

=> texlive-20200406_4: fetching distfile 'texlive-20200406-source.tar.xz'...

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

The build has been terminated
```

Any clue?

Reply:
Unsynced mirror, it's trying to build `texlive` in CI to fulfill the dependency. Not a problem.